### ISSUE TITLE
Can't scan with change to accounts.csv

### ISSUE THREAD

**Issue by eq2kazial** [Noctem/Monocle#156]
I'm now receiving the following error anytime I try to run scan.py with the update to using accounts.csv. I've included my worker accounts username/passwords/ptc in the csv but left all other fields blank.

```
Traceback` (most recent call last):
  File "scan.py", line 324, in <module>
    main()
  File "scan.py", line 309, in main
    overseer.start()
  File "/Users/Scott/Monocle/monocle/overseer.py", line 91, in start
    self.workers = tuple(Worker(worker_no=x) for x in range(self.count))
  File "/Users/Scott/Monocle/monocle/overseer.py", line 91, in <genexpr>
    self.workers = tuple(Worker(worker_no=x) for x in range(self.count))
  File "/Users/Scott/Monocle/monocle/worker.py", line 115, in __init__
    self.initialize_api()
  File "/Users/Scott/Monocle/monocle/worker.py", line 131, in initialize_api
    device_info = get_device_info(self.account)
  File "/Users/Scott/Monocle/monocle/utils.py", line 250, in get_device_info
    if account['iOS'].startswith('1'):
AttributeError: 'NoneType' object has no attribute 'startswith'
```

**Comment by GoldenMat** [comment]
Having the same exact issue

**Comment by GouSao** [comment]
dont delete the 6 columns in top

**Comment by GoldenMat** [comment]
Managed to fix it, for some reason the script doesn't like if the first letter of the username is in upper case. Just write it in lower case and it'll work just fine

**Comment by eq2kazial** [comment]
Fixed with commit 5ffe126e447ae820cd24533362781900fa47e48a
